x-pack/plugins/security_solution/public/timelines/components/side_panel/new_user_detail/mocks/index.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RiskSeverity } from '../../../../../../common/search_strategy';
```

Comment:
I just noticed that sometimes the mock folder is called `mocks` and other times `__mocks__`. I will rename it to `__mock__` because it is used more often.

@logeekal  Do you know what the difference is?

Reply:
so from `__mocks__` folder jest automatically picks up the mocks. It is a good way to create typesafe mocks. Here is the documentation: https://jestjs.io/docs/manual-mocks#mocking-user-modules.

But __mocks__ can be used to mock one module in one file.

Reply:
I didn't know that. It is very convenient. I will use it frequently. Thanks!